docs/changehistory/NextVersion.md

```diff
@@ -16,3 +16,16 @@ ignore: true
 ## Updated version of Electron
 
 Updated version of electron used from 8.2.1 to 10.1.3. Note that Electron is specified as a peer dependency in the iModel.js stack - so it's recommended but not mandatory that applications migrate to this electron version.
+
+## Globe location tool fixes
```

Comment:
Mention here the new `FromGcs` versions of various functions?

Reply:
Added